torch/_functorch/partitioners.py

```diff
@@ -491,10 +493,24 @@ def should_quantize(node: torch.fx.Node) -> bool:
 
     # calculate the size of the node
     size_in_mb = calculate_tensor_size(node.meta["val"])
-
-    return size_in_mb >= torch._inductor.config.post_grad_fusion_options[
+    if torch._inductor.config.post_grad_fusion_options[
```

Comment:
readability nit: would be easier if you factored out the threshold into a separate variable. especially since you use it in multiple places, might as well CSE here